pages/plugins.js

```diff
@@ -47,23 +47,23 @@ const Plugins = ({ plugins }) => {
               isLink={true}
               onClick={() => handleStartNowClick("PluginsLearnMoreClick")}
               className="!inline-block  !text-xs    !font-normal  !bg-transparent hover:!bg-transparent !px-2"
-              href="https://docs.amplication.com/plugins/overview/"
+              href="https://docs.amplication.com/getting-started/plugins/"
               delayLinkMs={300}
             />
             <Button
               text="Create your own plugin"
               isLink={true}
               onClick={() => handleStartNowClick("PluginsCreateYourOwnClick")}
               className="!inline-block  !text-xs    !font-normal  !bg-transparent hover:!bg-transparent !px-2"
-              href="https://docs.amplication.com/plugins/overview/"
+              href="https://docs.amplication.com/plugins/how-to-create-plugin/"
               delayLinkMs={300}
             />
             <Button
               text="Request a new plugin"
               isLink={true}
               onClick={() => handleStartNowClick("PluginsRequestNewClick")}
               className="!inline-block  !text-xs    !font-normal  !bg-transparent hover:!bg-transparent  !px-2"
-              href="https://docs.amplication.com/plugins/overview/"
+              href="https://docs.amplication.com/contributing/"
```

Comment:
let's use this URL instead 

https://github.com/amplication/amplication/issues/new?assignees=&labels=type%3A+feature+request&projects=&template=feature.yml